steps/bundler/0.0.3/step.yml

```diff
@@ -0,0 +1,37 @@
+title: Bundler
+summary: Install Ruby Dependencies with Bundler
+description: Install Ruby Dependencies with Bundler
+website: https://github.com/chrisfuller/bitrise-step-bundler
+source_code_url: https://github.com/chrisfuller/bitrise-step-bundler
+support_url: https://github.com/chrisfuller/bitrise-step-bundler/issues
+published_at: 2019-09-08T16:41:15.363368-07:00
+source:
+  git: https://github.com/chrisfuller/bitrise-step-bundler.git
+  commit: 5941e841a021216b90c13e32f1b38bfdc7ac6691
+host_os_tags:
+- osx
+- ubuntu
+type_tags:
+- utility
```

Comment:
pls remove `utility` type_tag in the next release, as this step handles dependencies. `utility` is used if no other category matches the step.

Reply:
Removed in `v0.0.4` 👍